WordPress/Classes/ViewRelated/Aztec/Helpers/PostEditorUtil.swift

```diff
@@ -0,0 +1,430 @@
+import Foundation
+
+typealias PostEditorViewControllerType = UIViewController & PublishablePostEditor
+
+class PostEditorUtil: NSObject {
```

Comment:
You're doing an awesome cleanup job in this PR.

With this specific class, though, I'm a bit confused about its purpose and scope.

I think having a very generic name ("Util") and a bit of a generic scope (UI, State handling, etc) can make it hard to figure out what goes where moving forward.

I have a sensation by looking at the code that we could split this class into (at least) two classes: one for UI (alerts) and another one for post state handling.

PS: the idea is not to hold this PR back though, but I wanted to share those thoughts with you.

Reply:
Thanks for checking this out!
The PR regarding this specific section was to enable `Publish` and `Save` functionality to Gutenberg Controller.
I share you concern but with @pinarol we decided to simplify the initial work.
You can see some notes about this here.
https://github.com/wordpress-mobile/WordPress-iOS/pull/10461#pullrequestreview-176444065
TL;DR:
We are aware of the mixing of concerns and we will take a closer look to this and how to solve it in the Beta timeline.

Reply:
That covers it for me, since it's known and a deliberate choice for now.  Thanks for the clarification.